lib/pages/fragments/SearchBar.dart

```diff
@@ -172,6 +211,7 @@ class SearchTile extends StatelessWidget {
             if (isHist && label != DEFAULT_LABEL)
               GestureDetector(
                 onTap: () {
+//                  LocalKeyValuePersistence.removeFromSearchHistory(label);
```

Comment:
Is this supposed to be commented out?

Reply:
Oups! Yes we can delete this comment. The action is taken inside the HistoryDeleteEvent which is also associated with the comment above.